repo/meta/index.json

```diff
@@ -244,6 +244,20 @@
       }
     },
     {
+      "currentVersion":"1.1.0-RC1",
+      "description":"A container orchestration platform for Mesos and DCOS.",
```

Comment:
We should add a warning saying that this pkg is for Marathon release candidates and shouldn't be used on production.